userdocs/src/usage/eks-private-cluster.md

```diff
@@ -60,6 +60,21 @@ privateCluster:
 
 The endpoints supported in `additionalEndpointServices` are `autoscaling`, `cloudformation` and `logs`.
 
+### Skipping endpoint creations
+
+If a VPC has already been created with the necessary AWS endpoints set up and linked to the subnets described in the EKS documentation,
+`eksctl` can skip creating them by providing the option `skipEndpointCreation` like this:
+
+```yaml
+privateCluster:
+  enabled: true
+  skipEndpointCreation: true
+```
+
+_Note_: this setting cannot be used together with `additionalEndpointServices`. It will skip all endpoint creation. Also, this setting is
+only recommended if the endpoint <-> subnet topology is correctly set up. I.e.: subnet ids are correct, vpne routing is set up with prefix addresses,
```

Comment:
`vpne` typo ?

Reply:
also capitalise if you are saying `VPN routing` :D

Reply:
Uh, yes. :D nice catch. was supposed to be vpce.